optionals/arcadian/CfgVehicles.hpp

```diff
@@ -0,0 +1,235 @@
+class CfgVehicles {
+
+class Car_F;
+class CUP_SUV_Base: Car_F { 
+    class HitPoints {
+        class HitLFWheel;
+        class HitLF2Wheel;
+        class HitRFWheel;
+        class HitRF2Wheel;
+        class HitEngine;
+        class HitFuel;
+        class HitBody;
+        class HitGlass1;
+        class HitGlass2;
+        class HitGlass3;
+        class HitGlass4;
+    };
+    
+    class NewTurret;
+    class Turrets {
+        class MainTurret: NewTurret {
+            class HitPoints;
+        };
+    };
+    class Eventhandlers;
+    class AnimationSources;
+};
+
+class CUP_SUV_Unarmed_Base: CUP_SUV_Base {
+    class AnimationSources: AnimationSources {
+        class rear_window_source {
+            source = "door";
+            initPhase = 0;
+            animPeriod = 1;
+        };
+    };
+};
+
+class CLASS(SUV_Base): CUP_SUV_Unarmed_Base {
+    armor = 90;
+    crewCrashProtection = 0.25;
+    displayName = "GT SUV Base";
+    hiddenSelections[] = { "camo" };
```

Comment:
Get rid of spaces in array definitions, everywhere.
```suggestion
    hiddenSelections[] = {"camo"};
```
Is this hidden selection not already defined in `CUP_SUV_Unarmed_Base`.